charter.md

```diff
@@ -32,18 +28,29 @@ The SPICE WG will profile existing IETF technologies and address residual gaps t
 
 The SPICE WG, coordinates with RATS, OAuth, JOSE, COSE and SCITT working groups that develop documents related to the identity and credential space. The SPICE WG builds on cryptographic primitives defined in the CFRG (e.g., BBS Signatures) and does not define novel cryptographic schemes.
 
-The SPICE WG will not develop digital credentials for any particular use case.  The general purpose profiles the WG will define will enable credential issuers to more easily build on existing IETF technologies.
+The SPICE WG will develop digital credential profiles which can support a number of use cases.
```

Comment:
Should the below be a list, supporting the use cases?

Reply:
No because we don't want to focus on specific use cases in the charter.